src/nvim/ex_cmds.c

```diff
@@ -5413,7 +5419,10 @@ helptags_one (
    */
   STRCPY(NameBuff, dir);
```

Comment:
Any instance of `STRCPY` looks suspicious to me, especially if it is not preceded (directly) with a length check. If I can replace `STRCPY` with anything I will do this, but specifically in this line there is something more efficient then `STRLCPY` to use. My variant creates a link between the code which defines `dirlen` (which has length check just in the next line) and this line and is faster then `STRLCPY`. If `dirlen` definition was located on other line it would at least need a comment about why such unsafe (and both `memcpy` and `STRCPY` are not safe) function is used.